packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -144,6 +146,50 @@ def test_if_we_have_capabilities(cluster):
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
 
 
+def test_cosmos_package_add(cluster):
+    def package_add_configured():
+        user_config = load_yaml("/opt/mesosphere/etc/user.config.yaml")
```

Comment:
The user config isn't what we want to load here. Ideally we want to say from outside the cluster that is running what state the cluster should be in for these conditional things. (See things like dns_search in https://github.com/dcos/dcos/blob/master/test_util/runner.py#L17). 

As a midway point, can for now probably load `/opt/mesosphere/etc/expanded.config.json` which has all the configuration values which were set or calculated. and grab the flags out of there. There might be a helper around for getting that file, talk to @mellenburg